.github/workflows/dev-service.yml

```diff
@@ -0,0 +1,77 @@
+name: Deploy Layer8 Server to Development Environment
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - development
+
+jobs:
+  build-layer8:
+    name: Build Layer8 Server
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: kciter/aws-ecr-action@v3
+        with:
+          access_key_id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          secret_access_key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          account_id: ${{ secrets.AWS_ACCOUNT_ID }}
+          repo: layer8
+          region: ${{ vars.AWS_REGION }}
+          tags: latest,${{ github.sha }}
+          create_repo: true
+  deploy-layer8-server-development:
+    name: "Deploy Layer 8 to Development"
+    environment: "development"
+    needs: [build-layer8]
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: infra/development/layer8
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: hashicorp/setup-terraform@v3
+        with:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+
+      - name: Install AWS CLI
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y awscli
+      - name: Populate Application Environment Variables to S3
+        id: populateappenv
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          AWS_DEFAULT_REGION: ${{ vars.AWS_REGION }}
+        run: |
+          echo -e "${{ vars.DEVELOPMENT_APP_ENV }}" >> app.env
+          aws s3 cp ./app.env s3://${{ vars.DEVELOPMENT_S3_APP_ENV_LOCATION }}
```

Comment:
In this code, we add all configuration or environment variable needed to run the layer8 server to the S3, we are using this to simplify the input of environment variable in the ECS task definition. The ECS will read all value from the S3 then put it to container environment variable automatically